framework/framework.cpp

```diff
@@ -274,6 +281,8 @@ Framework::Framework(const UString programName, const std::vector<UString> cmdli
 
 	this->threadPool.reset(new ThreadPool(threadPoolSize));
 
+	LogWarning(
+	    ("Current working directory: " + boost::filesystem::current_path().string()).c_str());
```

Comment:
...But it does happen to work in this case, as the const char[] at the beginning is constructed to a UString, then the operator+(const char*) called on that.

But anyway I prefer "%s" style for now, as we use it in most other places, and I like consistency.

Plus, GCC warns about non-const strings passed into printf strings (e.g if your path happened to contain a '%' printf specifier it'll probably do bad things